src/main/java/bio/terra/cloudfiletodatastore/deltalayer/functions/DeltaLayerRawFunction.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.cloudfiletodatastore.deltalayer.functions;
+
+import bio.terra.cloudfiletodatastore.FileCreatedMessageHarness;
+import bio.terra.cloudfiletodatastore.FileMessage;
+import bio.terra.cloudfiletodatastore.GsonConverter;
+import bio.terra.cloudfiletodatastore.deltalayer.DeltaLayerMessageProcessor;
+import com.google.cloud.functions.Context;
+import com.google.cloud.functions.RawBackgroundFunction;
+import com.google.events.cloud.storage.v1.StorageObjectData;
+import java.util.logging.Logger;
+
+public class DeltaLayerRawFunction
```

Comment:
This looks good too! If you want, we can also change `public abstract class CloudEventsHarness implements CloudEventsFunction` to `public abstract class CloudEventsHarness implements RawBackgroundFunction` so that `DeltaLayerRawFunction` can implement `CloudEventsHarness` directly and no need for the additional `FileCreatedMessageHarness`.